mlflow/store/sqlalchemy_store.py

```diff
@@ -277,13 +278,14 @@ def create_run(self, experiment_id, user_id, run_name, source_type, source_name,
                          start_time=start_time, end_time=None,
                          source_version=source_version, lifecycle_stage=LifecycleStage.ACTIVE)
 
+            tags_dict = {}
```

Comment:
Refactors this logic so that we don't attempt to insert the same tags twice (SQL DB doesn't like it because `tag_key, run_uuid` are supposed to be unique) e.g. if both `parent_run_id` and its corresponding tag are specified. As per the MlflowClient.create_run docs, the passed `parent_run_id` takes precedence over values in tags.